internal/netxlite/tlshandshaker_test.go

```diff
@@ -185,6 +185,9 @@ func TestUTLSHandshakerChrome(t *testing.T) {
 	}
 	cfg := &tls.Config{ServerName: "google.com"}
 	conn, err := net.Dial("tcp", "google.com:443")
+	if err != nil {
+		t.Fatal("unexpected error", err)
+	}
```

Comment:
Not strictly part of this PR, but `staticcheck` noticed it while I was opening files.